cgroup.go

```diff
@@ -207,7 +235,7 @@ func (c *cgroup) add(process Process, pType procType, subsystems ...Name) error
 		if err != nil {
 			return err
 		}
-		err = retryingWriteFile(
+		err = writeCgroupProcs(
```

Comment:
Do we still need the EINTR checks that retryingWriteFile does? In 1.15 Go handles this for us and will retry, but it was never back ported to 1.14 afaict where EINTR is more common as well. Not sure if we even use anything in this lib that would block someone on 1.14 from using it

Reply:
Oh nice, we use os.ReadFile and friends now from 1.16 https://github.com/containerd/cgroups/commit/98dc9d945cc4026ec441e3416a12c4899d45f0b4

Reply:
No. We can remove retryingWriteFile() now.

https://github.com/golang/go/issues/38033

Reply:
So truthfully we should be able to get rid of that retryingWriteFile throughout I believe. I can look into that

Reply:
Awesome. Seems like this pkg is 1.16 minimum because we got off of ioutil, and the eintr fix was in 1.15. I'll drop retryingWriteFile from the rest of the project